tests/integration/test_job_runner_mgr.py

```diff
@@ -82,3 +83,58 @@ async def test_kill_error(one, start, test_dir, capsys, log_filter):
             level=logging.WARNING,
         )
         assert itask.state(TASK_STATUS_RUNNING)
+
+
+async def test_create_nn_new(one, start, test_dir, capsys, log_filter):
+    """Test _create_nn. 
+
+    It should create the NN symlink.
+    """
+    async with start(one):
+        itask = one.pool.get_tasks()[0]
+
+        workflow_job_log_dir = Path(get_workflow_run_job_dir(one.workflow))
+        job_id = itask.tokens.duplicate(job='01').relative_id
+        job_log_dir = Path(workflow_job_log_dir, job_id)
+        job_log_dir.mkdir(parents=True)
+
+        # call _create_nn
+        JobRunnerManager()._create_nn(job_log_dir / 'job.out')
```

Comment:
I think this is usually the path to the job script
```suggestion
        JobRunnerManager()._create_nn(job_log_dir / 'job')
```